package.json

```diff
@@ -9,6 +9,7 @@
   ],
   "scripts": {
     "lint": "eslint packages --ext .js,.ts,.tsx",
+    "types:check": "tsc --skipLibCheck --noEmit",
```

Comment:
Putting the script here instead of running `tsc` directly from husky is what let me delete the config file with function syntax.